### ISSUE TITLE
[SIMPLE QUESTION]

### ISSUE THREAD

**Issue by RolandoAvides** [open-mmlab/OpenPCDet#483]
Hello,

Does the road_plane dataset help to improve a models performance?

Thanks in advance!

**Comment by jihanyang** [comment]
No significant influence. You can set `USE_ROAD_PLANE` to false and compare the results.

**Comment by RolandoAvides** [comment]
Thanks!

**Comment by RolandoAvides** [comment]
Hi, @jihanyang I recently sent you an e-mail with a few questions about Open-PCDet. I hope you could answer.

Thanks!

**Comment by Viczyf** [comment]
So what results did you get with and without use road_plane data? Do you mind to share it with me? Thanks. @RolandoAvides

**Comment by Viczyf** [comment]
> ![image](https://user-images.githubusercontent.com/21019429/130962625-18eef6da-3737-4304-85d7-b7a4dd61eb56.png)
> 
> As you can see in my experiments, the performance on val dataset is better and more stable if ROAD_PLANE is set to true.

Is this result in line with expectation? If I do not use road plane data, it's hard to achieve equal performance with that claimed in the paper.

**Comment by DeclK** [comment]
My results are very similar with @Viczyf
It seems this USE_ROAD_PLANE augmentation is really helpful in KITTI. I tried to train  Voxel R-CNN with smaller batches, because I only have 3 GPUs. But I can't reach the claimed results without USE_ROAD_PLANE. I tried batch size 12 & 8 & smaller learning rate, none of those combinations could reproduce the claimed performance.

I just tried USE_ROAD_PLANE with 12 batches for 120 epochs, and changed nothing else, the result is much better

Car AP@0.70, 0.70, 0.70:
bbox AP:96.8361, 89.7339, 89.3944
bev  AP:90.3240, 88.2947, 87.8128
3d   AP:89.3482, 84.0559, 78.7073
aos  AP:96.81, 89.64, 89.25

Car AP_R40@0.70, 0.70, 0.70:
bbox AP:98.3673, 94.9160, 94.5469
bev  AP:93.4076, 91.1892, 88.9424
3d   AP:92.2308, 84.9129, 82.5564
aos  AP:98.35, 94.79, 94.36
